builder.go

```diff
@@ -256,8 +281,15 @@ func (b *Builder) ToSQL() (string, []interface{}, error) {
 	if err := b.WriteTo(w); err != nil {
 		return "", nil, err
 	}
+	var sql = w.writer.String()
+	if b.dialect == POSTGRES {
```

Comment:
MsSql `@p1` `@p2` see https://github.com/denisenkom/go-mssqldb
Oracle `:1` `:2` see https://github.com/mattn/go-oci8/blob/master/_example/lastinsertid/main.go#L53

Reply:
Got it